### PR TITLE
Sort bedtools output in bam_to_bigwig conversion

### LINKED ISSUE #2582: BAM to BigWig fails for Trackster loading
I have some BAM files generated using hisat2 that I am trying to load against hg20 on our local (16.04) instance. After much hackery (effectively satisfying the galaxy_packages dependency resolver) I got ucsc_tools to be resolved but now I get an error from CONVERTER_bam_to_bigwig_0:

temp.bg is not case-sensitive sorted at line 18294564.  Please use "sort -k1,1 -k2,2n" with LC_COLLATE=C,  or bedSort and try again.

The BAM from hisat2 is already sorted (the wrapper ends with samtools sort) so I do not know where this error is coming from.

### PR DESCRIPTION
**Pull request by lparsons**
Bedtools genomecov seems to output bedgraph files with chromosomes
sorted in the order given in the chromInfo file. This can cause issues
with bedGraphToBigWig which requires them sorted in a specific way.

Fixes #2582.

### CODE REVIEW

File: lib/galaxy/datatypes/converters/bam_to_bigwig_converter.xml
Comment (nsoranzo) [inline lib/galaxy/datatypes/converters/bam_to_bigwig_converter.xml:14]: s/wigToBigWig/bedGraphToBigWig/ I think

**Comment by nsoranzo** [comment]
LGTM, ping @pvanheus.

File: lib/galaxy/datatypes/converters/bam_to_bigwig_converter.xml
Comment (lparsons) [reply lib/galaxy/datatypes/converters/bam_to_bigwig_converter.xml:14]: Actually, the line as is seems to work, so I'm inclined to leave it. `wigToBigWig claims to support bedGraph files (from help):

```
wigToBigWig v 4 - Convert ascii format wig file (in fixedStep, variableStep
or bedGraph format) to binary big wig format.
```

**Review by nsoranzo — APPROVED** [review]

**Comment by lparsons** [comment]
Can this get "backported" to `16.07`?

**Comment by nsoranzo** [comment]
@lparsons Since this has not been merged yet, you can try to change the target branch of this PR to `release_16.07` and see if it still applies cleanly.

**Comment by lparsons** [comment]
Ugh. That didn't seem to work as intended... I guess it's not too important if 16.10 is only days away, but wouldn't want this to be a bug for too much longer.

**Comment by nsoranzo** [comment]
OK, I should have known better, but we can still backport after this is merged. Let's ping again @pvanheus , which reported the original issue.

**Comment by lparsons** [comment]
@nsoranzo Actually, I found this issue independently and then saw it was reported, but happy to wait on @pvanheus to review.

**Comment by lparsons** [comment]
Can this get merged into release_16.07?

**Pull request merged**
